client/js/components/pages/dashboard.jsx

```diff
@@ -60,6 +60,16 @@ export default class Dashboard extends React.Component {
             </Card>
           </div>
         </div>
+        <div className='row' style={{marginBottom: '15px'}}>
+          <div className='col-xs-12 col-sm-12'>
```

Comment:
it doesn't take 100% width by default ;) more of some random value as it's not 50% as well.. but i'll leave just `col-xs-12` and delete `col-sm`